### DISCUSSION TITLE
Could not open configuration file:

### DISCUSSION THREAD

**Discussion by oakdeneal** [Motion-Project/motionplus#185] [Q&A]
all seemed to go ok until the last bit.
`sudo motionplus`
then received the following message:
ALR][ALL][mp00] init: Could not open configuration file:
I'm not 100% NOOB just 99.9%.

Operating System: Raspbian GNU/Linux 12 (bookworm)
Kernel: Linux 6.6.47+rpt-rpi-v6
Architecture: arm
Rpi camera

**Comment by gabbymouth** [comment]
You might want to try something like:
/usr/local/bin/motionplus/motionplus **-c /home/user-account/motionplus-conf-files/motionplus-dist.conf**

Also, sudo may not be needed.

**Reply by oakdeneal** [reply]
hi gabbymouth

had a look at that location and tried a few things.  it says motionplus is not a directory.  when I opened with `nano motionplus` _ File 'motionplus' is unwritable _ and the data looks unreadable.  first line reads(^?ELF^A^A^A^C^@^@^@^@^@^@^@^@^B^@(^@^A^@^@^@��^A^@4^@^@^@���^@^@^D^@^E4^@ ^@    ^@(^@(^@'^@^A^@^@ph^V   ^@h^V)
`pi@raspberrypi:/usr/local/bin $ ls
motionplus
pi@raspberrypi:/usr/local/bin $ motionplus
[ALR][ALL][mp00] init: Could not open configuration file: 
pi@raspberrypi:/usr/local/bin $ cd motionplus
-bash: cd: motionplus: Not a directory
`

**Comment by hormigaclub** [comment]
At the end explain "The configuration files"
https://motion-project.github.io/motionplus_config.html

El lun, 30 sept 2024 a la(s) 12:37 p.m., gabbymouth (
***@***.***) escribió:

> You might want to try something like:
> /usr/local/bin/motionplus/motionplus -c
> /home/user-account/motionplus-conf-files/motionplus-dist.conf
>
> Also, sudo may not be needed.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Motion-Project/motionplus/discussions/185#discussioncomment-10799799>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAR3YQGCA6MP4E4ECU5TIQDZZFVZZAVCNFSM6AAAAABPDSEXIWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZZHE3TSOI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <Motion-Project/motionplus/repo-discussions/185/comments/10799799@
> github.com>
>


-- 
Gustavo Javier Ovalle
Ituzaingo 1365
S3070BPW - San Cristóbal
Santa Fe
e-mail: ***@***.***

**Reply by oakdeneal** [reply]
Hi hormigaclub

thanks for the steer.
I looked at that, tried to invoke motionplus from all mentioned locations.

_Current directory where MotionPlus was invoked_  tried from there
`pi@raspberrypi:~/motionplus/data $ motionplus
[ALR][ALL][mp00] init: Could not open configuration file:`

The directory called '.motionplus' in the current users home directory_ tried from there
`pi@raspberrypi:~/motionplus $ motionplus
[ALR][ALL][mp00] init: Could not open configuration file:`

The directory defined by --sysconfdir=DIR during the configure step of installation. If this was not defined, it will default to /etc/motionplus_. No motionplus directory or file.
`pi@raspberrypi:/etc $ ls motionplus
ls: cannot access 'motionplus': No such file or directory`

I did look in usr/local/etc/motionplus_ tried here also and same result.
`pi@raspberrypi:/usr/local/etc/motionplus $ ls
camera1-dist.conf  camera2-dist.conf  camera3-dist.conf  motionplus-dist.conf  sound1-dist.conf
pi@raspberrypi:/usr/local/etc/motionplus $ motionplus
[ALR][ALL][mp00] init: Could not open configuration file:`

Could it be that I should have git cloned motionplus to root directory instead of home?

**Reply by smynard** [reply]
Did you work it out @oakdeneal? I'm having a similar issue. From what I can tell it looks like the motionplus.conf file isn't getting created.

**Reply by Mr-Dave** [reply]
Just edit the motionplus-dist.conf and camera1-dist.conf files to meet your needs and change the name of the files
